lgc/builder/BuilderImpl.h

```diff
@@ -295,7 +295,7 @@ class DescBuilder : virtual public BuilderImplBase {
 
   // Create a pointer to a descriptor.
   llvm::Value *CreateGetDescPtr(ResourceNodeType descType, unsigned descSet, unsigned binding,
-                                const llvm::Twine &instName) override final;
+                                ResourceNodeType resourceType, const llvm::Twine &instName) override final;
```

Comment:
I believe the arguments here should be renamed to conreteType and abstractType to match the change in ResourceNode from the other PR.